testscommon/cacherStub.go

```diff
@@ -1,21 +1,25 @@
 package testscommon
 
+import "github.com/multiversx/mx-chain-go/storage/txcache"
+
 // CacherStub -
 type CacherStub struct {
-	ClearCalled             func()
-	PutCalled               func(key []byte, value interface{}, sizeInBytes int) (evicted bool)
-	GetCalled               func(key []byte) (value interface{}, ok bool)
-	HasCalled               func(key []byte) bool
-	PeekCalled              func(key []byte) (value interface{}, ok bool)
-	HasOrAddCalled          func(key []byte, value interface{}, sizeInBytes int) (has, added bool)
-	RemoveCalled            func(key []byte)
-	RemoveOldestCalled      func()
-	KeysCalled              func() [][]byte
-	LenCalled               func() int
-	MaxSizeCalled           func() int
-	RegisterHandlerCalled   func(func(key []byte, value interface{}))
-	UnRegisterHandlerCalled func(id string)
-	CloseCalled             func() error
+	ClearCalled                           func()
+	PutCalled                             func(key []byte, value interface{}, sizeInBytes int) (evicted bool)
+	GetCalled                             func(key []byte) (value interface{}, ok bool)
+	HasCalled                             func(key []byte) bool
+	PeekCalled                            func(key []byte) (value interface{}, ok bool)
+	HasOrAddCalled                        func(key []byte, value interface{}, sizeInBytes int) (has, added bool)
+	RemoveCalled                          func(key []byte)
+	RemoveOldestCalled                    func()
+	KeysCalled                            func() [][]byte
+	LenCalled                             func() int
+	MaxSizeCalled                         func() int
+	RegisterHandlerCalled                 func(func(key []byte, value interface{}))
+	UnRegisterHandlerCalled               func(id string)
+	CloseCalled                           func() error
+	SelectTransactionsWithBandwidthCalled func(numRequested int, batchSizePerSender int, bandwidthPerSender uint64) []*txcache.WrappedTransaction
```

Comment:
this is no longer a CacherStub. I would create a new stub here like:
```
type TxCacherStub struct {
    *CacherStub
    ....
}
```

Reply:
Done